crates/biome_lsp/src/extension_settings.rs

```diff
@@ -28,6 +31,9 @@ pub struct WorkspaceSettings {
 pub struct ExperimentalSettings {
     /// Enable experimental symbol renaming
     pub rename: Option<bool>,
+
+    /// Path to the configuration file to prefer over the default `biome.json`.
+    pub configuration_path: Option<String>,
```

Comment:
I think it's safe to move this outside of the experimental flags. `rename` was experimental because... well, it's basically useless 😆